src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -1495,8 +1493,10 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
 
             string rootModule = null;
             if (
-                !GetScalarFromData<string>(data, moduleManifestPath, "RootModule", manifestProcessingFlags,
-                    out rootModule))
+                !GetScalarFromData<string>(
```

Comment:
this was just my OCD going crazy sorry

Reply:
Could you please make this in one line?
```c#
!GetScalarFromData<string>(data, moduleManifestPath, "RootModule", manifestProcessingFlags, out rootModule))

Reply:
I really think it looks better like this but sure